PWGHF/D2H/TableProducer/candidateCreatorCharmResoReduced.cxx

```diff
@@ -212,7 +219,7 @@ struct HfCandidateCreatorCharmResoReduced {
         }
         float invMassReso{0.};
         float invMassV0{0.};
-        std::array<float, 3> pVecV0 = candV0.pVector();
+        std::array<float, 3> pVecV0 = {candV0.px(), candV0.py(), candV0.pz()};
```

Comment:
Why are you reverting this?

Reply:
I deleted the PVector column for that table

Reply:
Why?

Reply:
I used dynamic columns for the candidate's momentum components, so I couldn't use them as argument of the PVector column anymore. I could have defined a new dynamic column that takes the daughter tracks' momenta as an input...